tests/test_game_logic.py

```diff
@@ -1,63 +1,59 @@
 import pytest
-from ..game_logic import check_bingo, manage_turns
-from ..bingo_card_generation import generate_bingo_card, display_bingo_card
-
-
+import game_logic
+import bingo_card_generation
+import user_interaction
 
 @pytest.fixture
 def setup_game():
     players = {
         "player1": {
-            "card": generate_bingo_card([f"Category {i}" for i in range(100)]),
+            "card": bingo_card_generation.generate_bingo_card([f"Category {i}" for i in range(100)]),
             "marked": [[False]*5 for _ in range(5)]
         }
     }
     player = "player1"
     return players, player
 
-
 def test_check_bingo():
     # Test row bingo
     marked = [[True, True, True, True, True],
               [False, False, False, False, False],
               [False, False, False, False, False],
               [False, False, False, False, False],
               [False, False, False, False, False]]
-    assert check_bingo(marked) == True, "Should detect a bingo in the first row"
+    assert game_logic.check_bingo(marked) == True, "Should detect a bingo in the first row"
 
     # Test column bingo
     marked = [[True, False, False, False, False],
               [True, False, False, False, False],
               [True, False, False, False, False],
               [True, False, False, False, False],
               [True, False, False, False, False]]
-    assert check_bingo(marked) == True, "Should detect a bingo in the first column"
+    assert game_logic.check_bingo(marked) == True, "Should detect a bingo in the first column"
 
     # Test no bingo
     marked = [[True, False, False, False, True],
               [False, True, False, True, False],
               [False, False, False, False, False],
               [False, True, False, True, False],
               [True, False, False, False, True]]
-    assert check_bingo(marked) == False, "Should not detect a bingo"
+    assert game_logic.check_bingo(marked) == False, "Should not detect a bingo"
 
 def test_manage_turns(monkeypatch, setup_game):
-    # Mocking dependencies and user input
-    from ..wikipedia_api import validate_category_from_url
-    from ..user_interaction import handle_user_input_with_timeout, mark_category
-    import save_data
-
-
     players, player = setup_game
 
-    def mock_display_bingo_card(card, marked):
+    def mock_input(prompt):
```

Comment:
The `handle_user_input_with_timeout` function from `user_interaction` makes a call to the builtin `input` function so we need to mock it.